.travis.yml

```diff
@@ -24,6 +24,10 @@ matrix:
       env:
         - LARAVEL_VERSION=^7.0
         - PHPUNIT_VERSION=^8.0
+    - php: "7.3"
```

Comment:
Add PHP 7.4

Reply:
I only added a single PHP 7.4 entry, for testing of latest versions of PHPUnit and Laravel. But let me know if you want any additional cases added to the test matrix. :v: